src/js/components/form-elements/v2/Checkbox.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import InputWrapper from 'wrappers/InputWrapper';
+
+import './style.scss';
+
+const CheckBox = ({
```

Comment:
the file is named "Checkbox", while the component is named "CheckBox" - was it meant to be?